fcrepo-event-serialization/src/main/java/org/fcrepo/event/serialization/JsonLDEventMessage.java

```diff
@@ -34,85 +33,29 @@
 
 /**
  * A structure used for serializing a FedoraEvent into JSON
+ * 
  * @author acoburn
+ * @author dbernstein
  */
 class JsonLDEventMessage {
 
     @JsonIgnore
     private static final Logger LOGGER = getLogger(JsonLDEventMessage.class);
 
-    static class ContextElement {
-
-        @JsonProperty("@id")
-        public final String id;
-
-        @JsonProperty("@type")
-        public final String type;
-
-        public ContextElement(final String id) {
-                this.id = id;
-                this.type = "@id";
-        }
-
-        public ContextElement(final String id, final String type) {
-            this.id = id;
-            this.type = type;
-        }
-    }
-
-    static class Context {
-        public final String prov = "http://www.w3.org/ns/prov#";
-
-        public final String foaf = "http://xmlns.com/foaf/0.1/";
-
-        public final String dcterms = "http://purl.org/dc/terms/";
-
-        public final String xsd = "http://www.w3.org/2001/XMLSchema#";
-
-        public final String type = "@type";
-
-        public final String id = "@id";
-
-        public final ContextElement name = new ContextElement("foaf:name", "xsd:string");
-
-        public final ContextElement identifier = new ContextElement("dcterms:identifier");
-
-        public final ContextElement isPartOf = new ContextElement("dcterms:isPartOf");
+    public static final String ACTIVITYSTREAMS_NS = "https://www.w3.org/ns/activitystreams";
 
-        public final ContextElement atTime = new ContextElement("prov:atTime", "xsd:dateTime");
+    public static final String USER_AGENT_BASE_URI_PROPERTY = "fcrepo.auth.webac.userAgent.baseUri";
+    static class Object {
 
-        public final ContextElement wasAttributedTo = new ContextElement("prov:wasAttributedTo");
-
-        public final ContextElement wasGeneratedBy = new ContextElement("prov:wasGeneratedBy");
-    }
-
-    static class Activity {
         @JsonProperty("type")
         public List<String> type;
 
-        @JsonProperty("identifier")
-        public String identifier;
+        @JsonProperty("id")
+        public String id;
 
-        @JsonProperty("atTime")
```

Comment:
Is there some way we can keep the timestamps in the message, to convey when the activity happened?  

There are some options in the [activityStreams vocabulary](https://www.w3.org/TR/activitystreams-vocabulary), but maybe we should bounce this off @escowles , to see if the spec should say something about it.

Reply:
I think we can use the https://www.w3.org/TR/activitystreams-vocabulary/#dfn-starttime and https://www.w3.org/TR/activitystreams-vocabulary/#dfn-endtime properties (possibly with the same value ).

Reply:
👍

Reply:
@escowles :  I'm leaning towards "published"  - seems more clear - the time the event was published (think pub/sub)  looks like endtime and starttime are more geared towards events that span time (e.g. social events).

Reply:
@dbernstein "published" seems like a better fit to the kind of data we have available.  The Fedora impl does not keep track of actions' duration, so to say that the internal timestamp Fedora is using equates to start/end times may be misleading.   This will allow us to later on add start and end times when/if it makes sense to do so (like for long-running operations where it might matter).

Reply:
@dbernstein Yes, "published" on the event object sounds like a better fit.